src/main/java/bhs/devilbotz/RobotContainer.java

```diff
@@ -67,6 +72,10 @@ public class RobotContainer {
 
   /** The container for the robot. Contains subsystems, OI devices, and commands. */
   public RobotContainer() {
+    if (Robot.isReal()) {
+      cameraManager = new CameraManager();
```

Comment:
Only create cameraManager on real robot. _This should probably be handed inside of CameraManager_.